.github/workflows/ci.yml

```diff
@@ -21,22 +21,9 @@ jobs:
     - name: Run Infrastructure tests on BouncyCastle
       run: |
         dotnet test tests/DotNetLightning.Infrastructure.Tests -p:BouncyCastle=True
-    - name: Restore NSec and Secp256k1.Net
-      run: git checkout src/NSec src/Secp256k1.Net
-    - name: Clean to prepare for NSec build
-      run: |
-        dotnet clean
-    - name: Run tests
-      run: |
-        dotnet test
-    - name: Package NSec build
-      run: |
-        cd $GITHUB_WORKSPACE/src/DotNetLightning.Core
-        dotnet pack -p:Configuration=Release -p:Version=1.1.0-date`date +%Y%m%d-%H%M`.git-`echo $GITHUB_SHA | cut -c 1-7`
-    - name: Upload nuget packages (BouncyCastle and native)
+    - name: Upload BouncyCastle nuget package
       run: |
         cd $GITHUB_WORKSPACE/src/DotNetLightning.Core
-        if [ ${{ secrets.NUGET_API_KEY }} ] && [ $GITHUB_REF == "refs/heads/master" ]; then
-            dotnet nuget push ./bin/Release/DotNetLightning.1*.nupkg -k ${{ secrets.NUGET_API_KEY }} -s https://api.nuget.org/v3/index.json
-            dotnet nuget push ./bin/Release/DotNetLightning.Core.1*.nupkg -k ${{ secrets.NUGET_API_KEY }} -s https://api.nuget.org/v3/index.json
+        if [ ${{ secrets.NUGET_API_KEY }} ]; then
```

Comment:
you need to restore the `&& [ $GITHUB_REF == "refs/heads/master" ]` bit here before we can merge this in our fork